### ISSUE TITLE
Escaping issue with (<50 character sequence (should render as text but instead it drops the rest of the message)

### ISSUE THREAD

**Issue by freelock** [element-hq/element-web#15651]
### Description

Escaping issue with the sequence `(<50` :

![image](https://user-images.githubusercontent.com/46937/98279552-f0a22380-1f4e-11eb-8f1a-b330f5477ac5.png)

### Steps to reproduce

This is a message that came through the Slack bridge, but it renders fine in Element Android -- it's just broken on Element web. You can see from the screenshot that no text is rendered after the "(" character -- the rest of the message is dropped, even though it is in the message source.

**Comment by t3chguy** [comment]
The escaping issue is in the slack bridge. That's not valid HTML.

**Comment by dbkr** [comment]
Yeah, we could switch to more of a tag-soup parser that will try & make sense of whatever nonsense you throw at it, but I don't think it would exactly be a priority and there's a certain argument to failing more obviously so we notice when bridges are misbehaving like this.

**Comment by dbkr** [comment]
Another option would be to ignore the formatted body completely if it's invalid HTML and use the plain body.

**Comment by t3chguy** [comment]
Closing as not an Element issue, this is following the Matrix spec just fine
